packages/ide-extension/src/extension.ts

```diff
@@ -19,7 +20,9 @@ export function activate(context: ExtensionContext): void {
                     ide: await getIde()
                 };
                 logString(`Guided Answers command called. Options: ${JSON.stringify(options)}`);
-                const guidedAnswersPanel = new GuidedAnswersPanel(options);
+                if (!guidedAnswersPanel) {
```

Comment:
@unseen1980, if you call Guided Answers command the second time with different options, I think the the new options would not be evaluated? I think calling Guided Answers command multiple times with different options (like initial visible tree and path) is a valid scenario and should be supported. Or did I miss something?